src/client/app/helpers/snippet.helper.ts

```diff
@@ -72,7 +73,6 @@ export const snippetFieldSortingOrder: { [key: string]: number } = {
 export function getSnippetDefaults(): ISnippet {
     return {
         id: '',
-        gist: '',
```

Comment:
Why did you remove the empty-string gist?  It seemed to be a reasonable default...